### DISCUSSION TITLE
Lesson 5-- Brownie  Error

### DISCUSSION THREAD

**Discussion by lthibeault** [smartcontractkit/full-blockchain-solidity-course-py#39] [Q&A] [answered]
I recieve the following error when trying to run a script in Brownie.

ModuleNotFoundError: No module named 'Crypto.tutorial'

Crypto.tutorial is the "folder Structure" but I am on Windows, and its not in my C: Drive, 
I have even tried giving it the full Directory of the script folder.

I
![Capture](https://user-images.githubusercontent.com/8378411/133790074-d13a14fc-f14e-45b5-b136-93d1d0f13045.PNG)

**Comment by PatrickAlphaC** [comment]
Thanks for making this question!

Could you send me what your:

1. Deploy script
2. Your file structure
3. Could you copy paste that error instead of using an image? 

Thank you!

**Reply by lthibeault** [reply]
Thank you for the assist, I have seen a few others on the internet have this same issue. 
I have noticed its looking for a Crypto.tutorial module...
That is the "Path" of the project directory (D:/Crypto/tutorial/project folder

**Comment by lthibeault** [comment]
The Error:

Brownie v1.16.3 - Python development framework for Ethereum

BrownieSimpleStorageProject is the active project.

Launching 'ganache-cli.cmd --port 8545 --gasLimit 12000000 --accounts 10 --hardfork istanbul --mnemonic brownie'...
  File "C:\Users\larry\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\_cli\run.py", line 49, in main
    return_value, frame = run(
  File "C:\Users\larry\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\project\scripts.py", line 53, in run
    module = _import_from_path(script)
  File "C:\Users\larry\.local\pipx\venvs\eth-brownie\lib\site-packages\brownie\project\scripts.py", line 149, in _import_from_path
    _import_cache[import_str] = importlib.import_module(import_str)
  File "C:\Python39\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
  File "<frozen, line line, in in
ModuleNotFoundError: No module named 'Crypto.tutorial'
Terminating local RPC client...

**Comment by lthibeault** [comment]
the deploy script contents:

def deploy_simple_storage():
    pass


def main():
    deploy_simple_storage()

**Reply by PatrickAlphaC** [reply]
Your deploy script is empty! Why are you using this instead of this:

https://github.com/PatrickAlphaC/brownie_simple_storage/blob/main/scripts/deploy.py

**Reply by lthibeault** [reply]
Because I am at the 4 hour and 35 minuite mark of the video.
it wont run a print(hello) in the script. I get the same error

**Reply by lthibeault** [reply]
I just tried to run the script in that deploy.py file as well, with the same error, its failing when trying to load the deploy.py file and "read it"

**Reply by lthibeault** [reply]
Its "fixed" I will just use my C: drive to write and store the code until I figure out how to get brownie to use my D: drive

**Comment by lthibeault** [comment]
my file structure of the project path:

D:\Crypto\tutorial\brownie_simple_storage>

**Comment by lthibeault** [comment, marked as answer]
I just created a folder on my C: drive, and the issue is "fixed" apparently Brownie is forcing me to use my C: Drive to store these files.

**Reply by CedricVo91** [reply]
for me it just helped to eliminate any "dots" in the folder name e.g. folder: "my.folder" to "myfolder"
